src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs

```diff
@@ -136,5 +140,17 @@ private ObjectFactory GetFactory()
                 return predicate(context, propertyName);
             };
         }
+
+        private static string[] SplitString(string original)
+        {
+            if (string.IsNullOrEmpty(original))
+            {
+                return new string[0];
+            }
+
+            var split = original.Split(',').Select(piece => piece.Trim()).Where(piece => !string.IsNullOrEmpty(piece));
+
+            return split.ToArray();
```

Comment:
A bit of a nitpick, but think about removing some of the ToArrays here by just adding to a list or using `SelectMany` in your top level loop.